screens/Home.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button, Alert, Image, TextInput, StackNavigator} from 'react-native';
+import { Font } from 'expo';
+
+export class HomeScreen extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./../assets/fonts/Double_Bubble_shadow.otf'),
+      'source-sans-pro': require('./../assets/fonts/SourceSansPro-Regular.ttf'),
+    });
+    this.setState({ fontLoaded: true});
+  }
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={{backgroundColor: '#C2515B', height: '80%'}}>
+          <View style={{flexDirection: 'row', paddingTop: 30}}>
+            <View style={styles.shadowPong}>
+              <View style={styles.pongContainer}>
+              {
+                this.state.fontLoaded ? (
+                  <Text style={styles.pongText}>Pong</Text>
+                ) : null
```

Comment:
If `this.state.fontLoaded` is ever `false` or `null`, no text will show up. Make sure that you always have text to render